app/gulp/js/questions.js

```diff
@@ -1,8 +1,14 @@
 // Code to control adding/removing answer choices
 $('body.questions.edit').ready(() => {
+  let num = 10000; // We only support 10000 answers per question, get over it
   $('#add-answer').click((e) => {
+    num -= 1;
     e.preventDefault();
-    $('#wrong-answers').append($('#new-answer').html());
+    let newAnswer = $('#wrong-answers div:first-child').prop('outerHTML');
+    newAnswer = newAnswer.replace(/_[0-9]+_/g, `_${num}_`);
+    newAnswer = newAnswer.replace(/\[[0-9]+\]/g, `[${num}]`);
+    newAnswer = newAnswer.replace(/value="*."/g, 'value=""');
+    $('#wrong-answers').append(newAnswer);
```

Comment:
It looks like this is getting the html from an example answer field, doing some regex stuff, and inserting a copy into the page. This works, but why not just insert the html literally (just start from a string and do some string concatenation)? I think that might be easier to understand than some regex stuff, plus I wouldn't need to look for `#wrong-answers div:first-child` in the page.